app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -43,13 +56,26 @@ public function store(Request $request): RedirectResponse
             'g-recaptcha-response' => app()->environment('production') ? ['required', new Recaptcha($request->ip())] : [],
         ]);
 
+        $request->session()->forget(['github_email', 'github_username']);
+
         $user = User::create([
             'name' => $request->name,
             'email' => $request->email,
             'username' => $request->username,
             'password' => Hash::make($request->string('password')->value()),
         ]);
 
+        if (is_string($githubUsername) && $githubUsername !== '') {
+            $errors = $gitHub->linkGitHubUser($githubUsername, $user, $request);
+            if ($errors !== []) {
+                $user->delete();
+
+                return to_route('register')->withErrors($errors, 'github');
+            }
+            $user->email_verified_at = Carbon::now();
```

Comment:
If the user registered using a GitHub account, we can mark the email as verified as GitHub will already have taken care of verifying this email